commons_ocollection/ograph_extended.ml

```diff
@@ -289,6 +289,22 @@ let generate_ograph_xxx g filename =
   ()
 
 
+let get_os =
+  let ic = Unix.open_process_in "uname" in
+  let uname = input_line ic in
+  let () = close_in ic in
+  match uname with
+  | "Darwin" -> `MacOs
+  | "Linux" -> `Linux
+  | _ -> `Unknown
+
+let launch_png_cmd filename =
+  let _status =
+    Unix.system (Printf.sprintf "dot -Tpng %s -o %s.png" filename filename) in
+  let _status =
+    Unix.system (Printf.sprintf "open %s.png" filename)
```

Comment:
does this call to 'open' work on Linux? In that case maybe the function should be called
launch_png_cmd_macos

Reply:
For me it would be fine if we just dumped the .png and I could open it manually.